### PR TITLE
P/brentpicasso/installer fix

### LINKED ISSUE #999: Windows installer doesn't work with Kivy 1.9.1

### PR DESCRIPTION
**Pull request by brentpicasso**
Update installer and instructions to reflect kivy 1.9.1. resolves #999

### CODE REVIEW

**Comment by ryandoherty** [comment]
PR NUMBER 1000!!!!

**Comment by brentpicasso** [comment]
illuminati confirmed

File: README.md [hunk 1]
```diff
@@ -75,24 +71,13 @@ The script will create the .App file and .dmg for distribution.
...
-## Creating installer for current version (Win7)
-
-1. Go into RaceCapture_App folder
-1. Activate the virtualenv: `_ve\Scripts\activate`
-1. `kivy` (to get paths set up)
-1. cd install
-1. `buildwininstall.bat` - deletes old build folders, builds installer and tries to run the package (to run manually use `dist\racecapture\racecapture`)
-
+1. Navigate command line to install subfolder of RaceCapture app directory
+1. build_win_release.bat <major>.<minor>.<bugfix>
```
Comment (ryandoherty) [inline README.md]: Why the addition of specifying the version #? Can we just pull it from main.py like other build tools?

Comment (brentpicasso) [inline README.md]: it's how it worked before. <rabble>

But that would be a good improvement.

Comment (ryandoherty) [inline README.md]: Oh, so just improving docs? If so, that's ok. :shipit:

Comment (brentpicasso) [inline README.md]: Well, I changed the spec file to actually work with Kivy 1.9.1 - essentially just to get us back to where we were when things were working.

**Pull request merged**